### PR TITLE
<views>Add hello world to html

### PR DESCRIPTION
**Pull request by Guavalines**
# What was the change?
sdajlksdjlaksdj

# Why this change?
jsdkjahkdjsh

# How to test it?
rails s 
be suere you install all the gems
be sure

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -14,6 +14,7 @@
   <body>
     <main class="container mx-auto mt-28 px-5 flex">
       <%= yield %>
+      <h1>Hello World!</h1>
     </main>
```
Comment (daniel-enqz) [inline app/views/layouts/application.html.erb:18]: Hey pana, good job, amazing code.
Lets just move the Hello world before the yield block

**Review by daniel-enqz — CHANGES_REQUESTED** [review]
Thanks pana jus some small changes

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (Guavalines) [reply app/views/layouts/application.html.erb:20]: The change was made!

Comment (Guavalines) [reply app/views/layouts/application.html.erb:20]: Added an extra space to HTML

Comment (Guavalines) [reply app/views/layouts/application.html.erb:20]: I did more work.

Comment (daniel-enqz) [reply app/views/layouts/application.html.erb:20]: Great pana lets merge this!

Comment (daniel-enqz) [reply app/views/layouts/application.html.erb:20]: bla bla bla

**Pull request merged**